src/rpc/net.cpp

```diff
@@ -1103,10 +1103,12 @@ UniValue AddrmanEntryToJSON(const AddrInfo& info)
     ret.pushKV("network", GetNetworkName(info.GetNetClass()));
     ret.pushKV("source", info.source.ToStringAddr());
     ret.pushKV("source_network", GetNetworkName(info.source.GetNetClass()));
+    ret.pushKV("mapped_as", connman.GetMappedAS(info));
+    ret.pushKV("source_mapped_as", connman.GetMappedAS(info.source));
```

Comment:
nit: I always found this confusing as I read `mapped as <something>` and not `mapped *A*utonomous*S*ystem` - don't have a concrete idea for a name and I guess it makes sense to use the same as in `getpeerinfo`.

Reply:
I wanted to keep same name as in `getpeerinfo` but I agree with you about the confusion, I do believe `mapped_asn` would be better because we show the AS number.

Reply:
I will leave as is for now.